roles/manage-gcp-infra/tasks/create-infrastructure.yaml

```diff
@@ -0,0 +1,45 @@
+---
+- name: setup gcloud configuration
+  shell: gcloud config configurations create openshift-{{ gcloud_env }}
+  register: result
+  failed_when: not ((result.rc == 0) or ("it already exists" in result.stderr))
+  changed_when: not ("it already exists" in result.stderr)    
+  
+- name: setup gcloud authenticationn
+  shell: gcloud auth activate-service-account --key-file {{ credentials_file }} --project {{ gcloud_project_name }}
+
+- name: create a temp directory fror the google templates
+  file:
+    path: /tmp/gcp_templates
+    state: directory
+    mode: 0755 
+
+- name: create the google template 
+  template:
+    src: ./templates/openshift-gcloud.yaml.j2
```

Comment:
No need to prefix with `./templates` as Ansible knows to look in the `templates` directory by default

Reply:
done